### ISSUE TITLE
The attribute `--service-name` is missing from the docs

### ISSUE THREAD

**Issue by Jandev** [Azure/azure-cli#17737]
I wanted to update the connection status of my Private Link Endpoint and found [a page in the docs where this has been described](https://docs.microsoft.com/en-us/azure/private-link/manage-private-endpoint?WT.mc_id=AZ-MVP-5003246#azure-cli).
It mentions a CLI command with the `--service-name`. From the docs it's not very clear what this should be, so I navigated to the docs for this command. However, this property isn't mentioned on the entire page.
When not using this argument/property, the following error is thrown:
> RequiredArgumentMissingError: the following arguments are required: --service-name

I'm using the following version of the cli.
```json
 pwsh> az version
{
  "azure-cli": "2.14.0",
  "azure-cli-core": "2.14.0",
  "azure-cli-telemetry": "1.0.6",
  "extensions": {
    "azure-devops": "0.18.0"
  }
}
```
It would be great if one of the documents can get an update.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 23e1b12b-8356-600c-ee13-64e75b186c1b
* Version Independent ID: 9ad46f78-0eb9-027a-e5db-9ab284f20788
* Content: [az network private-link-service](https://docs.microsoft.com/en-us/cli/azure/network/private-link-service?view=azure-cli-latest)
* Content Source: [latest/docs-ref-autogen/network/private-link-service.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/master/latest/docs-ref-autogen/network/private-link-service.yml)
* Service: **virtual-network**
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by yonzhan** [comment]
network

**Comment by msyyc** [comment]
@Jandev which cmd do you use for 'RequiredArgumentMissingError: the following arguments are required: --service-name'

**Comment by Jandev** [comment]
> @Jandev which cmd do you use for 'RequiredArgumentMissingError: the following arguments are required: --service-name'

At the moment I'm using `PowerShell 7.0.3`.
Starting up the Terminal, I see the following:
```powershell
PowerShell 7.0.3
Copyright (c) Microsoft Corporation. All rights reserved.

https://aka.ms/powershell
Type 'help' to get help.

   A new PowerShell stable release is available: v7.1.3
   Upgrade now, or check out the release page at:
     https://aka.ms/PowerShell-Release?tag=v7.1.3

**********************************************************************
** Visual Studio 2019 Developer Command Prompt v16.9.3
** Copyright (c) 2021 Microsoft Corporation
**********************************************************************
```

**Comment by kairu-ms** [comment]
The example in doc is not correct. This command is using '--name' instead of '--service-name' in CLI.

![image](https://user-images.githubusercontent.com/69238381/127084925-1b10e18f-76d3-4932-a380-9aee72bcce37.png)
